### PR TITLE
Flatten cell test

### PR DESCRIPTION
**Pull request by orenzohar**
Merge from https://github.com/demisto/content/pull/2314 to master

### CODE REVIEW

File: Scripts/script-CommonServerPython.yml [hunk 1]
```diff
@@ -529,9 +529,16 @@ script: |-
       if isinstance(data, STRING_TYPES):
           return data
```
Comment (anara123) [inline Scripts/script-CommonServerPython.yml]: what about the case if the data is NOT list, but string with special character in it?

**Comment by anara123** [comment]
@orenzohar your tests is failing in circle

File: Scripts/script-CommonServerPython.yml (hunk 1 above)
Comment (orenzohar) [reply Scripts/script-CommonServerPython.yml]: In that case there's no need to flatten anything, and the calling functions seems to know how to handle unicode.

**Comment by Itay4** [comment]
@orenzohar What is the status of this?

**Comment by anara123** [comment]
```
------ Test playbook: TestCommonPython start ------
TestCommonPython failed with error/s
Playbook TestCommonPython has failed:
	- Command: !TestPYCommonServer test_type="XML"
	- Body: Error from Scripts is : Script failed to run: 
Error: [Traceback (most recent call last):
   
   File "<string>", line 307
     demisto.results(toEntry('special character failure - flatten_cell))
                                                                       ^
 SyntaxError: EOL while scanning string literal
] (2604) (2603)
	- Body: Playbook **TestCommonPython** execution error
```

**Pull request merged**